odc/geo/_interop.py

```diff
@@ -43,6 +43,14 @@ def datacube(self) -> bool:
     def tifffile(self) -> bool:
         return self._check("tifffile")
 
+    @property
+    def azure(self) -> bool:
+        return self._check("azure.storage.blob")
+
+    @property
+    def s3(self) -> bool:
+        return self._check("boto3")
+
```

Comment:
- we use lower level `botocore` library
- property name is library name, not feature
- `s3 -> botocore` and `boto3 -> botocore`

Reply:
fair, changed that to botocore, noting that boto3 is in the requirements.